packages/admin-next/dashboard/src/routes/products/common/media-grid-view.tsx

```diff
@@ -0,0 +1,125 @@
+import { CheckMini, Spinner, ThumbnailBadge } from "@medusajs/icons"
```

Comment:
Please use the code standard we use across this project, components go in folder with the same name as component:

- /common
  - /media-grid-view
    - media-grid-view.tsx
    - index.ts

Reply:
@kasperkristensen sure 👍 I am curious though, why not just a flatter list of components? This just adds more nesting without a clear benefit